tests/SlugifyTest.php

```diff
@@ -45,6 +45,11 @@ public function setUp()
         $this->slugify = new Slugify([], $this->provider);
     }
 
+    public function tearDown()
+    {
+        Mockery::close();
```

Comment:
@seferov We don't need this here because the Mockery TestListeners is configured in the PHPUnit config ([see](https://github.com/cocur/slugify/blob/master/phpunit.xml.dist#L17))